wiki/Medals/Legacy_Beatmap_Packs/en.md

```diff
@@ -1,26 +1,20 @@
----
-needs_cleanup: true
----
-<!-- TODO: more like "history of beatmap packs"... this article shouldn't exist as-is -->
+# Legacy beatmap packs
 
-# Beatmap packs 0916
+Beatmap packs have been an integral part to the osu! community since its first introduction by ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) way back in 2008, in which some of the packs have even been made to be tied to certain [medals and achievements](/wiki/Medals) out there. However, as osu! and its community goes on to grow over the years, the playing/mapping meta at large has also been undergone a major shift and — after a considerable amount of demand from the public — [on 16 January 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) a major overhaul was finally made to a numerous amount of packs in order to keep the beatmap packs as a whole relevant with the times.
```

Comment:
```suggestion
Beatmap packs have been an integral part of the osu! community since their first introduction by ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) way back in 2008, in which some of the packs were tied to certain [achievements](/wiki/Medals) out there. However, as osu! and its community continued to grow over time, the mapping meta at large has also undergone a major shift and — after a considerable amount of demand from the public — [on 16 January 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) a major overhaul was finally made to numerous beatmap packs in order to keep them, as a whole, relevant with the times.
```